### DISCUSSION TITLE
Please add instruction how to build and install on OpenWRT

### DISCUSSION THREAD

**Discussion by denizzzka** [hufrea/byedpi#125] [Ideas]

**Comment by artemklevtsov** [comment]
Please see: https://github.com/spvkgn/ByeDPI-OpenWrt

**Comment by redarrow-rus** [comment]
It seems byedpi has compatibility problems with OpenWRT. For example, when testing connectivity with curl from OpenWRT, it fails. It doesn't matter where byedpi is installed: on remote server, on local workstation, on router itself. In all cases, byedpi fails to process requests from router and works properly when getting requests from any other places.

**Comment by redarrow-rus** [comment]
I'm sorry for mistake. OpenWRT curl is built without TLSv1.3 support which is critical for requesting via byedpi in some cases. This misled me.

**Comment by redarrow-rus** [comment]
I managed to build byedpi for OpenWRT using this manual: https://openwrt.org/docs/guide-developer/toolchain/crosscompile
